.gitignore

```diff
@@ -1,3 +1,5 @@
 vendor/
 composer.lock
 build/
+.idea
+*.iml
```

Comment:
-1 for that. these are not project-specific but system-specific. you should use a system-level ignore rule: https://help.github.com/articles/ignoring-files/